test/functional/fixtures/api/es-next/resize-window/test.js

```diff
@@ -0,0 +1,17 @@
+describe('[API] Resize window actions', function () {
+    describe('t.resizeWindow', function () {
+        it('Should resize the window', function () {
+            return runTests('./testcafe-fixtures/resize-window-test.js', 'Resize the window', { only: 'local' });
```

Comment:
Actually we need no for the `only: 'local'` option. Just place the tests into the condition:
```js
if(!config.isTravisTask) {
    describe(...)
}
```
like [here](https://github.com/DevExpress/testcafe/pull/566/files#diff-f2713ec67a2f50ae2c99459aa4b1fad6R8)